ClubMembersSortedCF/src/members/v1/PersonBuilder.java

```diff
@@ -0,0 +1,42 @@
+package members.v1;
+
+import java.io.IOException;
+import java.time.LocalDate;
+
+public class PersonBuilder {
+    private Person person = new Person();
+
+    PersonBuilder surname(String surname) {
+        person.setSurname(surname);
+        return this;
+    }
+
+    PersonBuilder firstName(String firstName) {
+        person.setFirstName(firstName);
+        return this;
+    }
+
+    PersonBuilder birthDate(LocalDate birthDate) {
+        person.setDateOfBirth(birthDate);
+        return this;
+    }
+
+    PersonBuilder residence(int zipcode, String settlement, String address) throws InvalidZipCodeException {
+        Residence residence = new Residence(zipcode, settlement, address);
+        person.setResidence(residence);
+        return this;
+    }
+
+    PersonBuilder enterDate(LocalDate enterDate) {
+        person.setDateOfEnter(enterDate);
+        return this;
+    }
+
+    PersonBuilder grade(Grade grade) {
+        person.setGrade(grade);
+        return this;
+    }
```

Comment:
The good, old Builder pattern - good job :) In `Lombok`, you can just use a `@Builder` annotation instead, but great practice indeed